package.json

```diff
@@ -88,7 +88,7 @@
   "dependencies": {
     "@appnest/masonry-layout": "^2.2.3",
     "@glidejs/glide": "^3.6.1",
-    "@stackla/handlebars": "^0.1.3",
+    "@stackla/handlebars": "^0.1.5",
```

Comment:
Did we introduce any hbs helpers specific to rendering short videos?

Reply:
Yes @manikandan-ravikumar , I created a new ifShortVideo hbs helper for that, you can view the details here https://github.com/Stackla/stackla-packages/pull/123/files